src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx

```diff
@@ -48,28 +49,50 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
 
   return (
     <div className="mt-4 flex flex-col items-center justify-center text-center">
-      <Image src="/assets/images/dancing.gif" width={218} height={218} />
-      <div className="pt-8 text-4xl font-medium">Congratulations!</div>
-      <div className="mt-6 text-xl font-normal">{deployGreeting}</div>
-      <div className="flex flex-col gap-2 py-12">
-        <Button
-          type="primary"
-          onClick={handleGoToProject}
-          loading={gotoProjectClicked}
-        >
-          <span>
-            <Trans>Go to project</Trans>
-          </span>
-        </Button>
+      <Image
+        src="/assets/images/create-success-hero.webp"
```

Comment:
**suggestion** if you import this image using a JS import (`import image from "/public/assets..."`), you don't need to define width/height (something i learned recently!)

Reply:
Done but is there any benefit in not using height and width props to downsize?

Reply:
Hows it look after running lighthouse?

Reply:
Actually not sure how to run lighthouse for something like this, where it's on the `/create` page but required a state change (project creation) to load. Is it possible?

Reply:
hmm that is a great point.